parm/config/gfs/config.base

```diff
@@ -509,6 +509,11 @@ if [[ ${DO_JEDIATMVAR} = "YES" ]]; then
    export DO_VERFOZN="NO"     # Ozone data assimilation monitoring
    export DO_VERFRAD="NO"     # Radiance data assimilation monitoring
    export DO_VMINMON="NO"     # GSI minimization monitoring
+   export DO_ANLSTAT="YES"    # JEDI-based analysis statistics
```

Comment:
Should we set `DO_ANLSTAT="NO" before this block to ensure `DO_ANLSTAT` is not undefined in non-JEDI runs?

Reply:
Good catch, Russ.